web/src/engine/websites/ComicWalker.ts

```diff
@@ -0,0 +1,113 @@
+import { Tags } from '../Tags';
+import icon from './ComicWalker.webp';
+import { type Chapter, DecoratableMangaScraper, type Manga, type MangaPlugin, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { Fetch, FetchCSS, FetchJSON, FetchRequest } from '../FetchProvider';
+import type { Priority } from '../taskpool/DeferredTask';
+
+type TEndpoints = {
+    nc: string,
+    cw: string
+}
+
+type APIPages = {
+    data: {
+        result: {
+            meta: {
+                source_url: string,
+                drm_hash: string
+            }
+        }[]
+    }
+}
+
+function ChapterExtractor(anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname + anchor.search,
+        title: anchor.title.trim()
+    };
+}
+
+@Common.MangaCSS(/^https?:\/\/comic-walker\.com\/contents\/detail\/[^/]+\/$/, 'div#mainContent div#detailIndex div.comicIndex-box h1')
+@Common.ChaptersSinglePageCSS('div#ulreversible ul#reversible li a', ChapterExtractor)
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('comicwalker', `コミックウォーカー (ComicWalker)`, 'https://comic-walker.com', Tags.Language.Multilingual, Tags.Source.Official, Tags.Media.Manga);
```

Comment:
Maybe add the three supported language tags instead of multilingual.

Reply:
I dont think its a good idea. Let's think further : about language media tagging. 

Websites have either one langage or multiple. For now if we except special website medias have no language tag.
Now how should we implement langage tag for media? 

Website has one langage tag? > Manga directly inherit the website langage tag, in constructor. A simple constructor modification.

Website has Multilingual tag? We can then put an "unknown" tag / manually taking care of the tag.

Website has multiple langage tag, what should we do? 

Maybe i ma overthinking here.

Reply:
Another idea : shouldnt we SEPARATE langage tags from the others since we put all kind of tags  in the same bag? So its easier when we have to look for langage tags only?

Reply:
Some websites support quite a large amount of different languages.
Multilingual was introduced because of laziness, think of it as kind like a bitmask:
`type Multilingual = EN | FR | ES | JP | ZH | ...`.
If it would be feasible, i would vote for the complete removal of multilingual.

Media must not inherit default language tags from their parent, they need them explicitly set by the website plugin. In case that no language tag can be determined, another solution must be applied e.g., like you suggested with `unknown` or don't adding a language tag at all.

However, i don't think we have enough resources to implement language extraction for 800+ websites, so it is something not happening in the near future...

Reply:
> Media must not inherit default language tags from their parent,
If website have one language tag i dont see why? Maybe you have a better understanding than me.

> However, i don't think we have enough resources to implement language extraction for 800+ websites, so it is something not happening in the near future...

If the inheritance from the parent would be ok for you it would already be way less work.

Reply:
I'm fine with inheriting the parents single language tag if it is beneficial. We probably may need to add more languages.

But personally I don't see a use case where a tag is really needed for every manga/chapter on a website that only provides a single language. Seems redundant to me, as the frontend could use the same logic to traverse the parents language tags on the fly.

Reply:
I agree its not really "to be used" in a single language website. Its all about consistency.
Its more like that we should do the same thing for every website, and langage tagging makes sense for multilingual websites. is the word.

If the frontend can accommodate  the lack of tag, fine for me 🦐